benchmark/scala/src/main/scala/com/microsoft/tpch/ConvertTpchCsvToParquetApp.scala

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+package com.microsoft.tpch
+
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{SaveMode, SparkSession}
+
+object ConvertTpchCsvToParquetApp {
+  def main(args: Array[String]): Unit = {
+    if (args.length != 2) {
+      println(
+        "Error: incorrect command to submit the application, please follow the instruction and specify the path to the source and destination directory as arguments."
```

Comment:
did you run scala-fmt on this? I think you need to turn off the formatting rule if you want to go over 120 max chars.
Also, let's just follow https://github.com/dotnet/spark/blob/master/benchmark/scala/src/main/scala/com/microsoft/tpch/App.scala for error message for consistency.

Reply:
I see. Somehow the formatting rule did not break the long string into 2 lines of strings in my IntelliJ, I manually made it two lines. Following the examples, the if loop goes like this:

```scala
    if (args.length != 2) {
      println("Usage:")
      println("\t<spark-submit> --master local[*] --class com.microsoft.tpch.ConvertTpchCsvToParquetApp microsoft-" +
        "spark-benchmark-<version>.jar")
      println("\t\t<path-to-source-directory-with-TPCH-tables> <path-to-destination-directory-to-save-parquet-file>")
      return
    }
```

@wentaowu Could you please make a change accordingly, thanks.